src/extensionApi.ts

```diff
@@ -190,7 +186,8 @@ export class CommandHandler {
     public async fullPublishServer(context?: Protocol.ServerState): Promise<Protocol.Status> {
         let serverId: string;
         if (context === undefined) {
-            return Promise.reject('Please select a server from the Servers view.');
+            serverId = await this.selectServer('Select runtime to publish');
```

Comment:
It would be preferable to be consistent in labels and use "Server" everywhere: 'Select Server to publish'

Reply:
Would you also like other traces of 'runtime' eradicated?
Like 'Download runtime?' or 'No, use runtime on disk'

Reply:
I'd agree since we'd preferrably be consistent to the user. Agree?

Reply:
Done deal